.github/workflows/e2e-build-images.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
           versions=""
           while IFS=$'\t' read -r version codename; do
               # Filter out unsupported LTS releases
-              if [[ "${codename}" =~ trusty|xenial|bionic ]]; then
+              if [[ "${codename}" =~ xenial|bionic|focal ]]; then
```

Comment:
Trusty still has "Legacy support" (not sure what exactly that is tbh): https://ubuntu.com/about/release-cycle

Should we keep testing it?

Reply:
The "legacy support" relates to the 2 years of additional support that we used to provide for some LTS'es before Ubuntu Pro was a thing.

We have stopped releasing adsys for versions older than Jammy for a while now, so it's not worth to test those anymore (and the package won't even build unless we backport golang-1.24 to those releases)